llvm/docs/KeyInstructionsDebugInfo.md

```diff
@@ -0,0 +1,114 @@
+# Key Instructions debug info in LLVM
+
+Key Instructions reduces the jumpiness of optimized code debug stepping. This document explains the feature and how it is implemented in LLVM. For Clang support please see the [Clang docs](../../clang/docs/KeyInstructionsClang.md)
+
+## Status
+
+In development, but mostly complete. The feature is currently disabled for coroutines.
```

Comment:
Perhaps briefly explain why it's disabled for coroutines...? Seems a bit random without any explanation.